cigaradvisor/scripts/linking-data.js

```diff
@@ -86,7 +86,7 @@ export default function addLinkingData() {
   addOrg(document.querySelector('head'));
   if (window.location.pathname === '/cigaradvisor') {
     addBlogPosts();
-  } else if (window.location.pathname.match(/\/cigaradvisor\/category\/[^/]+$/)) {
+  } else if (window.location.pathname.match(/\/cigaradvisor\/[^/]+$/)) {
```

Comment:
I don't think we can just assume this is true. Category paths were known to be below category. Now there are pages at this level of the content hierarchy which are not category pages. We likely need a constant hard-coded list of categories and their paths.

Reply:
Good point.

> We likely need a constant hard-coded list of categories and their paths.

In `utils.js` or better ?